npe2/manifest/commands.py

```diff
@@ -1,3 +1,4 @@
+from textwrap import dedent
```

Comment:
should textwrap get added to the dependencies?

Reply:
no that stdlib : https://docs.python.org/3/library/textwrap.html

Reply:
yay